tests/scripts/run-globbing.sh

```diff
@@ -0,0 +1,12 @@
+#!/bin/bash
+
+mkdir ./globbing
+cp tests/trx2junit.Tests/data/* ./globbing
+
+echo ""
+trx2junit ./globbing/*.trx
+
+echo ""
+./verify-xml.sh "multiple-args/mstest.xml"
+./verify-xml.sh "multiple-args/mstest-warning.xml"
+./verify-xml.sh "multiple-args/nunit.xml"
```

Comment:
Is this the correct path? Shouldn't this be `globbing`?
And all files should be validated, now they're more than at the time this test was written.